doc/CrossCompilation.md

```diff
@@ -39,4 +39,4 @@ This process is currently happened in different places for different platforms:
 
 1. For Android, this happened in `hermes/android/build.gradle`
 2. For Apple platforms, this happend in `hermes/utils/build-apple-framework.sh`
-3. For Emscripten, you can find an example from the `test-emscripte` job from `hermes/.circleci/config.yml`. Also see more details at [Building with Emscripten](../emscripten)
+3. For Emscripten, you can find an example from the `test-emscripte` job from `hermes/.circleci/config.yml`. Also see more details at [Building with Emscripten](./Emscripten.md)
```

Comment:
I'm guessing this is better because it works on both github.com and hermes website (with Docusaurus replacing the link to the correct url)?

Reply:
Docusaurus has a broken links checker now and this URL was reported as broken.

In general, I think it's a bad idea to link using relative URL paths, because it makes the link sensitive to your host provider adding an extra slash at the end of an URL or not (it is not something very std and lead to a ton of reported problems in all Jamstack frameworks)

Reply:
(and yes, it also works on Github 👍 )

Reply:
Oh thanks for providing that context. I certainly fought with trailing slash appended by github pages before: https://twitter.com/Huxpro/status/798816417097224193